src/components/FullPageMobileWrapper/FullPageMobileWrapper.scss

```diff
@@ -36,4 +36,8 @@
         padding: 2rem;
         border-top: 2px solid #f2f3f4;
     }
+
+    @include mobile {
+        height: calc(100vh - 7.5rem);
+    }
 }
```

Comment:
there are some cases where we need the styles to be affected specifically for mobile, this will override the above mobile-or-tablet mixin it is mobile view